src/app/features/home/post-capture-tab/post-capture-tab.component.scss

```diff
@@ -13,3 +13,107 @@ mat-grid-tile {
     border-radius: 4px;
   }
 }
+
+.sub-content {
+  padding-top: 60px;
+}
+
+.tab-content-post {
+  virtual-scroller {
+    width: 100vw;
+    height: 93vh;
+    top: 7vh;
+  }
+
+  app-post-capture-card {
+    width: 100%;
+  }
+}
+
+.capture-container {
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.set-container {
```

Comment:
IMHO, `series-container` might be a better naming without introducing another term, _set_.